integr8ly/tutorial-web-app
# INTLY-3688: Implement Solution page for new home page

## Motivation
See https://issues.jboss.org/browse/INTLY-3688 for details.

## What
Create a tabbed dashboard design and add the All Solution Patterns page as the first phase for the redesign. Add the solution pattern cards much like current design, remove the resources panel, bring the repository and authored info up a level from the walkthrough description to the main dashboard, change solution patterns count to a badge, and add new styling to accommodate the new design.
 
## Why
Design request.

## Verification Steps
1. Go to the main dashboard page (opens by default).
2. All services tab should display TBD by default - will be complete in a future implementation.
3. Click the All Solution Patterns tab.
4. Verify that the solution patterns display, and can be followed as normal.
5. Verify that the authored count is correct, and displays whether it is Red Hat or Community accurately.
6. Verify that the Repository link is working and takes you to the corresponding repo for that group of solution patterns.
7. Verify that there are no apparent regressions or console errors/existing functionality is not broken.

Note that this is an ongoing effort and this is not to be merged to master until all changes are complete. Instead, changes will be staged in the homePageRedesign branch of tutorial-web-app.

## Checklist:

## Progress

## Additional Notes
**Screen cap**:
![all-solutions-tab](https://user-images.githubusercontent.com/39063664/68137081-5c911c80-fef4-11e9-9a25-322251d12153.png)

To test the new functionality live, feel free to use my cluster:
https://tutorial-web-app-webapp.apps.uxddev-e546.open.redhat.com/
admin@example.com / F1vrqCCy6lJVlCD

Or if you prefer, you can point to my public docker image from your own cluster:
docker.io/mfrances17/dev-tutorial-web-app:latest
